x-pack/platform/plugins/shared/maps/ui_tests/tests/full_screen_mode.spec.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect, tags, test } from '@kbn/scout';
+
+test.describe('Maps full screen mode', { tag: tags.DEPLOYMENT_AGNOSTIC }, () => {
+  const visibleChrome = 'kbnAppWrapper visibleChrome';
+  const hiddenChrome = 'kbnAppWrapper hiddenChrome';
+  test.beforeEach(async ({ browserAuth, pageObjects }) => {
+    await browserAuth.loginAsAdmin(); // add layer button not there when logged in as viewer
+    await pageObjects.maps.goto();
+  });
+  test('Full screen button should be visisble', async ({ page }) => {
+    const sel = 'mapsFullScreenMode';
+    await expect(
+      page.testSubj.locator(sel),
+      `Could not find the Full screen button, using selector ${sel}`
+    ).toBeVisible();
```

Comment:
I think we are good with basic error message like `Full screen button should be displayed in Map editor`, Scout will provide locator on its own:
```
Error: No results message in Saved Search panel should be visible

Timed out 10000ms waiting for expect(locator).toBeVisible()

Locator: locator('[data-test-subj="embeddedSavedSearchDocTable"]').locator('.euiDataGrid__noResults')
Expected: visible
Received: 
Call log:
  - No results message in Saved Search panel should be visible with timeout 10000ms
  - waiting for locator('[data-test-subj="embeddedSavedSearchDocTable"]').locator('.euiDataGrid__noResults')

    at x-pack/platform/plugins/private/discover_enhanced/ui_tests/tests/saved_searches.spec.ts:81:7
```